### DISCUSSION TITLE
6.4.6

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#597] [公告]
本版本有望成为 6.4.x 的稳定版本。如果没有任何实质性的重大更改或错误，PBH-BTN 倾向于在一定期限内暂停滚动发布以降低用户更新负担。

## 改进

* 优化了 50MB 以下的种子文件的反吸血措施
* 快速反作弊检测现在为 BitComet 用户自动关闭，直至 BitComet 补全所需的 API
* Portable 也更新到 Java 23，缓解 selector manager closed 的问题，请注意替换除了 `data` 文件夹内的所有文件
* 默认最大内存已提升到 512MB 以增强稳定性，无需担心内存使用变大，其中不使用的内存将被释放回系统

## 错误修复

* 修复 BiglyBT 出现错误的任务时，PBH 可能报错获取 Peers 时的 404 错误

## Docker 用户
DockerHub: `ghostchu/peerbanhelper:v6.4.6`
阿里云国内镜像加速: `registry.cn-hangzhou.aliyuncs.com/ghostchu/peerbanhelper:v6.4.6`

## 尾言

在今天（2024/10/13）的凌晨时分，我们发现有用户在 [BTN-Collected-Rules#24](https://github.com/PBH-BTN/BTN-Collected-Rules/issues/24) 发布充满恶意的言论。PBH-BTN 鼓励用户使用正当手段表达自己的不满，通过滥骂和问候家人并非适当的反馈手段，并可能导致问题被关闭和可能的法律后果。由于该言论下出现演变为滥骂战和教唆实施电信犯罪的情况，我们不得不立刻锁定了话题并折叠了有关评论以保护双方。  
本版发布时，已对主要仓库的 Issue 模板做出了修改，以便提醒通过恶意手段进行人身攻击和其它行为时 PBH-BTN 可能会做出的行动。您将在打开新的 Issue 时看到这些更改。请特别注意的是，这些更改不仅仅适用于 Issues，它还适用于所有由 PBH-BTN 直接管理的平台和服务。

我们再次公共地回答一些问题：我们的主要目标是打击无限制滥用 BT 网络的行为，而很多 PCDN 用户（或者服务提供商、公司）已经在大规模滥用 BT 网络大量吸血，其中不乏 PCDN 公司在其边缘节点内嵌 BT 刷流服务的情况。  
PCDN 大类不是 PBH-BTN 打击的目标，但是伤害 BitTorrent 网络环境的 PCDN，已经触碰到了每一个在如今网络环境下仍然愿意发挥互联网精神、无私奉献的用户。  

在 PBH 项目开展的数个月来，我们亲眼目睹和感受了大量这些**通过牺牲他人来谋取私利**的事情发生。受害者包括核心开发者在内的大量普通宽带用户，被这些谋取私利的行为引发了包括但不限于：

* 宽带限速（或进入黑名单 IP 段），失去传家宝套餐
* 上门检查
* 签署保证书
* 停机
* 拆机
* 被 ISP 拉黑

等种种恶劣后果。

是我们年入百万吃饱了没事干吗？不是，是有人**在通过伤害我们的方式**，谋取那不值一提的私利。  
这个组织完全是自发建立的公益组织，这也意味着一定有一股力量让我们走在了一起——不再被别人当作肥美的绵羊，合法合理的阻止自身合法利益再次受到损害。

我们会继续努力前行。即便我们倒下了，我们留下的公共代码资产也可以被任何人轻易的重建一套完全相同的反吸血的世界。这不是因为利益的关系，而是出于每个人心中正义的反抗。

您可以 Star、Fork 或者将存储库克隆到本地，来保护这份代码资产。或者加入我们，一起参与到反吸血的前线。

PBH-BTN / 2024-10-13

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.4.6'>6.4.6</a>.</em>

**Comment by poplar-at-twilight** [comment]
> 默认最大内存已提升到 512MB 以增强稳定性，无需担心内存使用变大，其中不使用的内存将被释放回系统

请问手动部署 PBH 需要把内存也设置为 512MiB 吗？

**Comment by Clackz** [comment]
真好笑，那群人干着损人利己的事，别人自保反而成了他们攻击别人的借口

**Comment by Huiaei** [comment]
吓得我赶紧建了个gitea仓库镜像

**Comment by Misaka-10034** [comment]
**前排支持！**
~已经是后排了~

**Comment by 6qs2w9oxhina** [comment]
支持，希望海南省的PCDN用户早日被清退。真不是只有你缺钱，想挣额外的钱，也请通过利人利己的方式。
搞PCDN的内鬼，boy鲁墨买墨吉！

**Comment by Unihis** [comment]
虽然不知道发生了什么，但支持，狠狠的支持。贡献自己的微薄之力

**Comment by fengqingxuezzz** [comment]
有isp内鬼存在的话这场战争注定会持续很久，如同疫情一般总会有放下口罩的那天吧，感谢这些防护者为我们的BT环境默默奉献着
